src/Ext/WarheadType/Detonate.cpp

```diff
@@ -36,6 +38,27 @@ void WarheadTypeExt::ExtData::Detonate(TechnoClass* pOwner, HouseClass* pHouse,
 
 		if (this->TransactMoney)
 			pHouse->TransactMoney(this->TransactMoney);
+
+		if (this->SpawnSuperWeapons.HasValue()) {
+			for (const auto pSWType : this->SpawnSuperWeapons.GetElements()) {
```

Comment:
After changing it to ValueableVector, here only needs
```suggestion
		for (const auto pSWType : this->SpawnSuperWeapons)
		{
```